Example/Backpack/View Controllers/IconsViewController.swift

```diff
@@ -0,0 +1,85 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import Backpack
+
+class IconsViewController: UICollectionViewController {
+    
+    fileprivate static var headingText = "All icons"
+    fileprivate static var icons = Array(Backpack.Icon.iconMapping!.keys).map{$0 as! String}.sorted(by: {$0 < $1})
```

Comment:
I feel like @k0nserv will know a better way of doing this. I'm converting `[ AnyHashable: Any ]` to `[ string ]` and then sorting it alphabetically.

Reply:
I think the code is fairly fine, but if the type of `iconMapping` in Swift is `Dictionary<AnyHashable, Any>` something isn't right. I'd expect it to have enough type information from Objective-c to deduce `Dictionary<String, String>`

Reply:
Yeah good point - do you have any pointers (pun intended) on where I can look to see about setting that in Obj-C? Can I pass types into the `NSDictionary` like `NSDictionary<string, string>` or something?